.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: Feature request
-about: Do not make a new issue for feature requests! Instead, post it on FeatHub, see the README.
+about: Do not make a new issue for feature requests! Instead, post it on the suggestions repository, see the README.
 
 ---
 
-**Do not make a new issue for feature requests!** Instead, post it on FeatHub: https://feathub.com/Anuken/Mindustry
+**Do not make a new issue for feature requests!** Instead, post it on this repository: https://github.com/Anuken/Mindustry-Suggestions
```

Comment:
> "this repository"

usage of the word **this** might confuse some people that don't know english very well 🤔

Reply:
oh yes
i suppose it's like desu